packages/office-ui-fabric-react/src/components/Button/Button.visualtest.ts

```diff
@@ -54,11 +68,16 @@ componentIds.push({
   commands: [defaultScreenshot]
 });
 componentIds.push({
+  selector: '#' + 'CompoundButtonToggled',
```

Comment:
Do you need to run a command here to update the baseline screenshots for visual tests?